### PR TITLE
Sync Localazy strings.

### PR DESCRIPTION
**Pull request by bmarty**
Replaces #2410

### CODE REVIEW

File: features/createroom/impl/src/main/res/values-be/translations.xml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   <string name="screen_create_room_error_creating_room">"Пры стварэнні пакоя адбылася памылка"</string>
   <string name="screen_create_room_private_option_description">"Паведамленні ў гэтым пакоі зашыфраваны. Гэта шыфраванне нельга адключыць."</string>
   <string name="screen_create_room_private_option_title">"Прыватны пакой (толькі па запрашэнні)"</string>
   <string name="screen_create_room_public_option_description">"Паведамленні не зашыфраваны, і кожны можа іх прачытаць. Вы можаце ўключыць шыфраванне пазней."</string>
   <string name="screen_create_room_public_option_title">"Адкрыты пакой (для ўсіх)"</string>
   <string name="screen_create_room_topic_label">"Тэма (неабавязкова)"</string>
   <string name="screen_start_chat_error_starting_chat">"Пры спробе пачаць чат адбылася памылка"</string>
+  <string name="screen_create_room_action_create_room">"Новы пакой"</string>
+  <string name="screen_create_room_action_invite_people">"Запрасіце сяброў у Element"</string>
+  <string name="screen_create_room_add_people_title">"Запрасіць карыстальникаў"</string>
```
Comment (bmarty) [inline features/createroom/impl/src/main/res/values-be/translations.xml:12]: I do not know why there is a reordering of keys, this is quite annoying. We may want to apply a post-treatment to order keys alphabetically...

Comment (bmarty) [reply features/createroom/impl/src/main/res/values-be/translations.xml:12]: Localazy is supposed to order keys alphabetically, this is maybe a regression on their side?

**Review by jmartinesp — APPROVED** [review]
Instant LGTM

**Comment by bmarty** [comment]
Thanks @jmartinesp for the screenshot recording. I did not see that the `All Chats` string has been changed to `Chats`. I have update the Maestro test according to that.

**Comment by bmarty** [comment]
@jmartinesp I have added a post treatment script to sort string resources in https://github.com/element-hq/element-x-android/pull/2436/commits/917fc12c2607ba9858e163b9c86d763cbc7280a4.

Hopefully it will work on the CI as well (we can wait on Monday to see it).

**Pull request merged**